undertow/src/main/java/org/wildfly/extension/undertow/ApplicationSecurityDomainDefinition.java

```diff
@@ -323,6 +324,7 @@ private static HttpAuthenticationFactory toHttpAuthenticationFactory(final Secur
                 new AggregateServerMechanismFactory(new BasicMechanismFactory(), new BearerMechanismFactory(),
                         new ClientCertMechanismFactory(), new DigestMechanismFactory(), new ExternalMechanismFactory(),
                         new FormMechanismFactory(), new SpnegoMechanismFactory()), SERVLET_MECHANISM);
+        ROOT_LOGGER.debugf("Security realm applied is \"%s\"", realmName);
```

Comment:
For debugging, I think it would be more useful to log the `securityDomain` here.